docs/contributing-guide.md

```diff
@@ -1,40 +1,28 @@
 
-- Before starting your work, ensure an [issue](https://github.com/chatwoot/chatwoot/issues) exist for it. If not feel free to create one. You can also take a look into the issues tagged [Good first issues](https://github.com/chatwoot/chatwoot/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22).
-- Add a comment on the issue and wait for the issue to be assigned before you start working on it.
-  - This helps to avoid multiple people working on similar issues.
-- If the solution is complex, propose the solution on the issue and wait for one of the core contributors to approve before going into the implementation.
-  - This helps in shorter turn around times in merging PRs
-- For new feature requests, Provide a convincing reason to add this feature. Real-life business use-cases will be super helpful.
-- Feel free to join our [discord community](https://discord.gg/cJXdrwS), if you need further discussions with the core team.
+2. Add a comment to the issue and wait for the issue to be assigned before you can start working on it. This helps avoid multiple people working on similar issues.
 
-### Pull Request Guidelines
+3. For complex solutions, propose the solution on the issue and wait for one of the core contributors to approve it before it can be implemented. This helps in shorter turnaround time in merging pull requests (PRs).
 
-- Please make sure that you have read the [issue triage guidelines](/docs/handbook/engineering/issue-triage) before you make a contribution.
-- We use [git-flow](https://nvie.com/posts/a-successful-git-branching-model/) branching model. The base branch is `develop`
-- Please raise your PRs against `develop` branch
-- It's okay and encouraged to have multiple small commits as you work on the PR - we will squash the commits before merging.
-- For other guidelines, please refer to [PR Guidelines](/docs/handbook/engineering/pr-guidelines)
-- Ensure that all the text copies that you add into the product are i18n translatable. You are only required to add the `English` version of the strings. We pull in the other language translations from our contributors on crowdin. Ref [Translation guidelines](https://www.chatwoot.com/docs/contributing-guide/translation-guidelines) to learn more.
+4. Raise your PRs against the base branch `develop`. We use the [git-flow](https://nvie.com/posts/a-successful-git-branching-model/) branching model.
+Multiple smaller commits are encouraged as you work on a PR as they will be squashed before merging.        
+- If you're resolving an issue, name the PR title as `fix: Fixes <issue-id>` and provide a detailed description of the bug in the PR. If applicable, also add appropriate test coverage.
+- For new feature requests, provide a convincing reason to add them; real-life business use cases will be super helpful. Create a branch in this format: `feature/<issue-id>-<issue-name>` (for example, `feature/235-contact-panel`) and add accompanying test cases.
```

Comment:
Not able to provide a sub-bullet here. I just end up getting a code block when I hit Tab.